hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/ExecutorException.java

```diff
@@ -0,0 +1,15 @@
+package org.apache.mesos.hdfs.executor;
+
+/**
+ * A invalid condition exist within the executor
+ *
+ * @author kensipe
+ */
```

Comment:
We should follow the same pattern as the ConfigurationException. with constructors for passing the message, cause, message + cause, and no empty constructor.